src/rest/APIRequest.js

```diff
@@ -6,7 +6,9 @@ const AbortController = require('abort-controller');
 const fetch = require('node-fetch');
 const { UserAgent } = require('../util/Constants');
 
-if (https.Agent) var agent = new https.Agent({ keepAlive: true });
+let agent;
+
+if (https.Agent) agent = new https.Agent({ keepAlive: true });
```

Comment:
```suggestion
const agent = new https.Agent({ keepAlive: true });
```
When would this class not exist?

The import could also be refactored to `const { Agent } = require('https');`

Reply:
Most likely a webpack remnant, which we removed a few months ago.